### PR TITLE
feat: Upgrade Messaging to new design.

### PR DESCRIPTION
**Pull request by defectiveAi**
- New design for communication between domains as collaborated across teams.
- Previous code was not used across teams (and after repos were archived, at all) has been removed as deprecated.

### CODE REVIEW

**Review by michaelmajgaard — APPROVED** [review]

File: source/Messaging/documents/release-notes/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Messaging Release notes
 
+## Version 3.0.0
+
+- Redesigned messaging library.
```
Comment (BjarkeMeier) [inline source/Messaging/documents/release-notes/release-notes.md:5]: How about a little more info? This is major new stuff and a severely breaking change.

**Review by BjarkeMeier — DISMISSED** [review]
LGTM

**Review by dstenroejl — DISMISSED** [review]
Workflow looks good to me

File: source/Messaging/documents/release-notes/release-notes.md (hunk 1 above)
Comment (defectiveAi) [reply source/Messaging/documents/release-notes/release-notes.md]: Updated.

**Review by dstenroejl — APPROVED** [review]
Approved workflow and csproj file changes

**Pull request merged**